### DISCUSSION TITLE
Buddha Quotes

### DISCUSSION THREAD

**Discussion by russellbanks** [maxkeppeler/sheets#34] [Show and tell]
#### We have implemented sheets 2.0.0 into our app, [Buddha Quotes](https://github.com/BanDev/Buddha-Quotes). It utilises Options, Input, Color and Time.

This is an Options Bottom Sheet that allows the user to share a quote or add it to a list.

<img src="https://user-images.githubusercontent.com/74878137/104209598-d8a4bf80-5429-11eb-899e-eac975c40e77.jpg" width="180"/>

This is an Input Bottom Sheet that allows the user to create a new list.

<img src="https://user-images.githubusercontent.com/74878137/104209599-d93d5600-5429-11eb-849b-0fdb098e7ddf.jpg" width="180"/>

This is a Time Bottom Sheet that allows for an input of time which then becomes a timer for meditation.

<img src="https://user-images.githubusercontent.com/74878137/104209600-d9d5ec80-5429-11eb-9b85-0ff6ca50582f.jpg" width="180"/>

This is an Options Dialog Sheet that allows the user to change the language of the app.

<img src="https://user-images.githubusercontent.com/74878137/104209601-d9d5ec80-5429-11eb-886c-7183dbac3ef9.jpg" width="180"/>

This is an Options Dialog Sheet that allows the user to change the theme of the app.

<img src="https://user-images.githubusercontent.com/74878137/104209603-da6e8300-5429-11eb-95b3-687c28953721.jpg" width="180"/>

This is a Color Dialog Sheet that allows the user to change the color accent of the app.

<img src="https://user-images.githubusercontent.com/74878137/104209605-da6e8300-5429-11eb-9edc-6ba5ad7d9c48.jpg" width="180"/>

**Comment by maxkeppeler** [comment]
Thanks for letting me know! It's great to see someone using my library, especially since I thought nobody would need it and I just created it for my own projects, originally!
